airbyte-integrations/connectors/destination-teradata/src/test-integration/java/io/airbyte/integrations/destination/teradata/TeradataDestinationSSLAcceptanceTest.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.teradata;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class TeradataDestinationSSLAcceptanceTest extends TeradataDestinationAcceptanceTest {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(TeradataDestinationSSLAcceptanceTest.class);
+
+  protected  String getConfigFileName() {
```

Comment:
this method is not being called anywhere as far as I can see, which means that this test runs w/o SSL